com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/utils/WorkspaceFileService.java

```diff
@@ -86,4 +87,16 @@ public Path getPathFromURI(@Nonnull String uri) {
       return null;
     }
   }
+
+  /**
+   * This method validates an URI to verify if is defined as extended document, according to a
+   * regex.
+   *
+   * @param uri document URI opened in the client
+   * @return true if the document is an extended document, false otherwise
+   */
+  public static boolean isFileUnderExtendedSourceFolder(String uri) {
```

Comment:
Isn't it overriding of an interface method?

Reply:
No but is a intermediate step to transform this class in an utility class later on!

Reply:
I cannot say that I like it, because, you know, there is nothing more permanent than a temporary solution. But I'll accept it